### DISCUSSION TITLE
Regrouping BT-763 and BT-31/BT-33

### DISCUSSION THREAD

**Discussion by MiraRatajczak** [OP-TED/eForms-SDK#686] [Ideas]
Why are BT-763 and BT-31/BT-33 in a separate group in the sdk (opposite of the eForms regulation)? Is it not logic that the fields are together? And should there be a rule that says if BT-763 is filled in then BT-31 is forbidden.

**Comment by rouschr** [comment]
Just note to myself: there is another notice-type related issue/idea in the discussions: https://github.com/OP-TED/eForms-SDK/discussions/517

**Comment by rouschr** [comment]
Please use the github discussions for subjects which are more suggestions, questions and ideas than proven technical bugs.

**Comment by rouschr** [comment]
According to Enrico this is a good idea, but currently we have too many other priorities.
We will keep this issue / idea open for later review.

**Comment by bertrand-lorentz** [comment]
As this is an idea/suggestion, I'm converting it to a discussion.

**Comment by mdewinne** [comment]
Bumping this ticket, we're still very much in favor of having a solution for this :-) 

In our application we now have to use little pointer texts to tell users about BT-31 "further down below":

![image](https://github.com/user-attachments/assets/0cf55d9f-1e6b-4e9b-a8c0-aa2646a9df98)
